static/js/app.js

```diff
@@ -152,6 +152,19 @@ function toggleVisibilityOfElementsIfAddonIsInstalled() {
 	}
 }
 
+async function addEmailToWaitlist(e) {
+  e.preventDefault();
+  const waitlistForm = e.target;
+  const formData = {
+    "csrfmiddlewaretoken": waitlistForm[0].value,
+    "email": waitlistForm[1].value
+  };
+  const response = await sendForm(waitlistForm.action, formData);
+  // So far, always a Bad Request
+	return;
```

Comment:
@groovecoder Was there some incantation for handling the alias forwarding preference form this way? I can't remember.